_events/2021/2021-02-career-panel.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "ADSA/US-RSE Early Career Panel"
+expires: 2021-02-23
+event_date: "February 23, 2021"
+layout: event
+repeated: false
+---
+
+We've teamed up with [The Academic Data Science Alliance (ADSA](https://academicdatascience.org/) to host an Early Career Panel on Research Software and Data Science careers in academia on February 23, 2021 at 2ET/11PT.  
+The event is primarily focused on students of all levels, postdocs, and those early in their careers, but all are welcome to attend.
```

Comment:
Is it focused on data science, or data science and research software engineering?

Reply:
The latter, careers as RSEs and DSs. I realize I forgot Engineering.